examples/07_router/src/entries.tsx

```diff
@@ -10,87 +10,97 @@ const BarPage = lazy(() => import('./components/BarPage'));
 const NestedBazPage = lazy(() => import('./components/NestedBazPage'));
 const NestedQuxPage = lazy(() => import('./components/NestedQuxPage'));
 
-export default createPages(async ({ createPage, createLayout }) => {
+const router = createPages(async ({ createPage, createLayout }) => {
   createLayout({
     render: 'static',
     path: '/',
     component: HomeLayout,
   });
 
-  createPage({
-    render: 'static',
-    // render: 'dynamic',
-    path: '/',
-    component: HomePage,
-  });
+  return [
+    createPage({
+      render: 'static',
+      // render: 'dynamic',
+      path: '/',
+      component: HomePage,
+    }),
```

Comment:
Out of curiosity, can we return createLayout too?
```js
const router = createPages(async ({ createPage, createLayout }) => [
  createLayout(...),
  createPages(...),
];
```
Not a strong opinion, but just feels a little bit consistent.

Reply:
We definitely can! We wouldn't do anything with them so there is no need to, but I also like that example you shared. It has nice visual consistency. 

Would you like me to change to that?

Reply:
Yeah, it seems cleaner.

Reply:
Updated!